docs/pages-for-subheaders/aws-marketplace-payg-integration.md

```diff
@@ -0,0 +1,273 @@
+
+Documentation can be found at the following link: https://ranchermanager.docs.rancher.com/getting-started/installation-andupgrade/install-upgrade-on-a-kubernetes-cluster/upgrades
+
+### Technical (Product)
+
+#### How do I get support?
+
+It is very simple to open a support case with SUSE for Rancher Prime. Create a ‘supportconfig’ via the Rancher UI and upload the output to the SUSE Customer Center. The support config bundle can be exported from the Rancher console using the ‘Get Support’ button at the bottom of the page. For deployments when Rancher is managing multiple downstream clusters, export the support config bundle from the primary cluster only.
+
+https://scc.suse.com/cloudsupport
+
+If the billing mechanism on the primary cluster is active, a support case will be opened. Further details can be found in the Rancher documentation: https://ranchermanager.docs.rancher.com/integrations-in-rancher/cloudmarketplace/supportconfig
+
+#### What are the resource requirements for installing Rancher on EKS?
+
+Please check the official documentation for best practices deployments.
+
+#### Is there any difference between Rancher Prime from the AWS Marketplace and the versions I can run in my own data center?
+
+Rancher Prime available in the AWS Marketplace is the same product, with the same functionality that you would run on-premises or with a manual installation. The only difference between deploying manually and deploying via the AWS Marketplace listing is the billing route.
+
+#### Does the primary cluster (responsible for billing) need to run 24/7?
+
+To ensure continuity with support, it is recommended that the primary Rancher cluster always remains active.
+
+#### What if the primary cluster responsible for billing is unable to connect to the AWS Billing framework?
+
+There may be multiple reasons why the primary cluster is unable to connect to the AWS framework, but it is the customer’s responsibility to ensure that the primary cluster is active and connected. Whilst the cluster is not connected to the billing framework, it is not possible to raise a support request.
+
+#### My primary cluster has been offline, what will happen with billing when reconnected?
+
+If the primary cluster is offline or disconnected from the AWS billing framework for a period of time, when it reconnects, the stored usage data will be uploaded to AWS and will appear on your next AWS bill.
+
+Depending on when in the month the primary cluster gets reconnected you may have several months of usage on your next billing cycle.
+
+#### Can the managed worker nodes reside on premises, at the edge or even on another cloud provider?
+
+Yes, nodes can run anywhere. SUSE Rancher will count the total number of nodes managed regardless of where they are deployed.
+
+#### How do I get fixes and updates for Rancher?
+
+To update the Rancher product to a current version before the marketplace listing is updated, please see the documentation.
```

Comment:
I'm not sure they can do this... @chajain @guangyee can you confirm? I assume to update they would need to switch to pulling from some other registry, which would risk unintentionally uninstalling the billing adapter, which would render the customer out of support

on a separate note, the link below 404's for me